app/models/miq_expression/component/is.rb

```diff
@@ -0,0 +1,17 @@
+class MiqExpression::Component::Is < MiqExpression::Component::Leaf
+  def to_arel
+    if target.date?
+      if MiqExpression::RelativeDatetime.relative?(value)
+        start_val = MiqExpression::RelativeDatetime.normalize(value, "UTC", _mode = "beginning").to_date
+        end_val = MiqExpression::RelativeDatetime.normalize(value, "UTC", _mode = "end").to_date
+        target.between(start_val..end_val)
+      else
+        target.eq(MiqExpression::RelativeDatetime.normalize(value, "UTC", _mode = "beginning").to_date)
+      end
+    else
+      start_val = MiqExpression::RelativeDatetime.normalize(value, timezone, _mode = "beginning").utc
```

Comment:
reason to include `_mode` here instead of just passing `"beginning"`?
(this is in a number of places)